sql/modules/Company.sql

```diff
@@ -79,7 +79,8 @@ create type eca_history_result as (
    serialnumber text,
    exchangerate numeric,
    salesperson_id int,
-   salesperson_name text
+   salesperson_name text,
+   transdate date
```

Comment:
There's just one nit with this PR and its that this change to the function signature requires dropping the function of the old signature: if we don't, we end up with two functions of the same name, only discriminated by the argument list. That's fine by me, but confuses the hell out of PGObject :-)
Could you please add the required DROP statement so I can merge the PR?

Thanks!

Reply:
Will do.  Thanks!